### ISSUE TITLE
Week 13, Part 1

### ISSUE THREAD

**Issue by deleonl4** [susanBuck/e2-fall20#100]
Hi all, my webpage doesn't show up when I type in e2p3.loc in the search bar. 
I tried troubleshooting and found that week 9, part 4 shows the example of `composer create-project --prefer-dist susanBuck/e2framework p3` in this case, being run in the ssh root for production. As the example on week 13 instructs, I ran it local but it isn't working for me. A screenshot shows the steps I took below. Any errors that I'm missing? I re-started Apache twice and checked that the code matched.

<img width="712" alt="Screen Shot 2020-12-05 at 10 11 22 PM" src="https://user-images.githubusercontent.com/47702390/101270445-10dc2280-3747-11eb-9b43-14ca53a1647c.png">

<img width="666" alt="Screen Shot 2020-12-05 at 10 11 29 PM" src="https://user-images.githubusercontent.com/47702390/101270446-12a5e600-3747-11eb-84d7-239b427264a0.png">

<img width="784" alt="Screen Shot 2020-12-05 at 10 14 46 PM" src="https://user-images.githubusercontent.com/47702390/101270468-5ac50880-3747-11eb-9920-85c9747853e1.png">

**Comment by deleonl4** [comment]
I'm also getting the following error when running the migration. I'm not sure if they're related because of permissions.

<img width="1149" alt="Screen Shot 2020-12-06 at 8 34 43 AM" src="https://user-images.githubusercontent.com/47702390/101281619-0731d980-379e-11eb-99f2-ffb42901fde5.png">

**Comment by susanBuck** [comment]
Hi @deleonl4 -

> my webpage doesn't show up when I type in e2p3.loc in the search bar.

Can you share a screenshot of what you see?

Can you also share the contents of your VirtualHosts config file (`/Applications/XAMPP/etc/extra/httpd-vhosts.conf`) and your hosts file (`/private/etc/hosts`)?

**Re: the migration error:**
It looks like your `.env` file is missing the following two configs:

```
DB_USERNAME='root'
DB_PASSWORD=''
```

**Comment by deleonl4** [comment]
@susanBuck It goes to a google search. 

<img width="1243" alt="Screen Shot 2020-12-06 at 9 11 45 AM" src="https://user-images.githubusercontent.com/47702390/101282431-2848f900-37a3-11eb-902b-cbafa464057b.png">

Thank you for the .env note. For some reason, as you highlighted those two lines in the video, I thought you deleted them when you mentioned something about them being local. I added them back in! 

<img width="1163" alt="Screen Shot 2020-12-06 at 9 41 20 AM" src="https://user-images.githubusercontent.com/47702390/101283266-46b0f380-37a7-11eb-9f60-73f012e56f49.png">

<hr>

<img width="1163" alt="Screen Shot 2020-12-06 at 9 41 08 AM" src="https://user-images.githubusercontent.com/47702390/101283267-47498a00-37a7-11eb-9f8e-141f943ddd2c.png">

**Comment by susanBuck** [comment]
Oh, you have to prefix the URL with `http://` when trying to enter it in the URL bar...e.g. `http://e2p3.loc`.

The reason is because Chrome does not recognize `.loc` as a real domain extension so it does know you are typing in a URL. Instead, it just interprets it as a search term, hence the Google search results.

**Comment by deleonl4** [comment]
Thank you @susanBuck! Unfortunately, now I have a new error :(

<img width="1237" alt="Screen Shot 2020-12-06 at 9 50 43 AM" src="https://user-images.githubusercontent.com/47702390/101283472-92b06800-37a8-11eb-9df3-00eaa0d6a62e.png">

**Comment by susanBuck** [comment]
New errors mean progress! : )

The error you're seeing is an Apache-based error (we see mention of Apache in the footer), which means we can assume that the e2p3 domain is correctly resolving to your local server... i.e. the changes to your hosts file were done correctly/are working.

In short, we can eliminate the hosts file step as a potential problem/cause.

Knowing this, we should focus on the VirtualHosts file as the potential culprit.

Looking at your screenshot, your VirtualHosts config file _looks_ okay to me, however, there's always the possibility there's some small character that's out of place that we're not seeing. Knowing this, can you copy/paste the actual text content of the VirtualHosts file and share it here? That way I can copy/paste it and test it directly on my end. 

Additionally, can you run the command `/Applications/XAMPP/bin/httpd -S` and share the output? This will reveal some debugging information about your VirtualHosts file. ([ref](https://hesweb.dev/e2/notes/local-server/local-domains#apache-wont-start-after-editing-virtualhosts-file-codehttpd-vhostsconfcode))

Finally - You've probably already done this (possibly a few times), but just to be sure, please restart Apache via your XAMPP control panel to make sure any config changes are taking effect.

**Comment by deleonl4** [comment]
@susanBuck True! Glass half full :)

 Thank you so much for your detailed observations/notes. 
I found the error after running the `/Applications/XAMPP/bin/httpd -S` command. 
I had a closing `</VirtualHost>` tag before starting the new `<VirtualHost>` tag. Copy and paste error. Thanks again!
